docs/conf.py

```diff
@@ -20,7 +20,7 @@
 # directory, add these directories to sys.path here. If the directory is
 # relative to the documentation root, use os.path.abspath to make it
 # absolute, like shown here.
-#sys.path.insert(0, os.path.abspath('.'))
```

Comment:
Are there any meaningful changes in this file? If not, I think that it should be left as it was.